src/main/scala/fi/oph/koski/valpas/fixture/ValpasDatabaseFixtureCreator.scala

```diff
@@ -0,0 +1,80 @@
+package fi.oph.koski.valpas.fixture
+
+import fi.oph.koski.config.KoskiApplication
+import fi.oph.koski.db.PostgresDriverWithJsonSupport.api._
+import fi.oph.koski.db.Tables._
+import fi.oph.koski.db._
+import fi.oph.koski.fixture.KoskiDatabaseFixtureCreator
+import fi.oph.koski.henkilo.{MockOppijat, OppijaHenkilö, VerifiedHenkilöOid}
+import fi.oph.koski.json.JsonSerializer
+import fi.oph.koski.perustiedot.{OpiskeluoikeudenOsittaisetTiedot, OpiskeluoikeudenPerustiedot}
+import fi.oph.koski.schema._
+import fi.oph.koski.valpas.henkilo.ValpasMockOppijat
+import slick.dbio.DBIO
+
+class ValpasDatabaseFixtureCreator(application: KoskiApplication) extends KoskiDatabaseFixtureCreator(application) {
```

Comment:
Tässä on aika paljon duplikaattikoodia KoskiSpecificDatabaseFixtureCreator:in kanssa, yhdistä.